src/quantem/tomography/tomography_base.py

```diff
@@ -338,20 +422,20 @@ def plot_projections(
             fig, ax = plt.subplots(ncols=3, figsize=(25, 8))
 
             show_2d(
-                np.abs(np.log(np.fft.fftshift(np.fft.fftn(self.recon_volume.array.sum(axis=0))))),
```

Comment:
I strongly dislike default log scale plotting!  This is why show_2d supports log and power law scaling.  If you make this function:
.plot_projections(
    ...
    fourier_transform=True,
    **kwargs,
)
you can pass **kwargs through to show (with defaults you want to specify, removed with "pop").